README.md

```diff
@@ -31,24 +31,42 @@ Making this more efficient and performant is what [IntersectionObserver](https:/
 
 ## Install
 
-Get the library using one of the following ways:
+This project uses [node](http://nodejs.org) and [npm](https://npmjs.com). Go check them out if you don't have them locally installed.
```

Comment:
@thiamsantos please don't create these many links, it distracts the user

Reply:
remove all the extra links added in README

Reply:
Why do you think that? I don't think that it has too much information, just what the user needs to getting started.

Reply:
> In other words, the more hyperlinks that you embed within your sentences, the less readable your posts become because the brain must make a decision with each link whether to click it for more information or keep reading. After several of these links, your brain starts to take on more cognitive load. As a result, it's easier to get sidetracked with tangents or to lose retention of the content.

Reference: http://idratherbewriting.com/2010/06/19/finally-convinced-about-removing-inline-links-to-increase-readability/

Reply:
Makes senses. Interesting article. So what do you think about change this part to something like:

```sh
# You can install lozad with npm
$ npm install --save lozad

# Alternatively you can use Yarn.
$ yarn add lozad

# Another option is to use Bower.
$ bower install lozad
```

Reply:
works

Reply:
done